tasks/updateconstants.js

```diff
@@ -89,13 +89,20 @@ const sources = [
         items[key.replace(/^item_/, '')] = item;
       });
 
-      // Load recipes
+    // Load recipes
       Object.keys(scripts).filter(key => scripts[key].ItemRequirements && scripts[key].ItemResult).forEach(key => {
         result_key = scripts[key].ItemResult.replace(/^item_/, '');
         items[result_key].components = scripts[key].ItemRequirements[0].split(";").map(item => item.replace(/^item_/, ''));
         items[result_key].created = true;
       });
 
+      //Manually Adding DiffBlade2
+      var manualDiffBlade2 = {};
```

Comment:
You can just copy the entire JSON blob for this item from the history and assign it to items['diffusal_blade_2'].  No need to assign things one by one.